src/common/cuepoint/cuepoint-manager.ts

```diff
@@ -0,0 +1,113 @@
+import { FakeEvent, PKTextTrack, EventType, TimedMetadata, createTextTrackCue, createTimedMetadata } from '@playkit-js/playkit-js';
```

Comment:
file marked as "created" instead of "renamed", will git save history in such case?

Reply:
yes the history will be saved in anyway but as another file